metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/query/ESBrowseDAO.java

```diff
@@ -133,14 +136,18 @@ public BrowseResult browse(@Nonnull String entityName, @Nonnull String path, @Nu
    */
   @Nonnull
   private AggregationBuilder buildAggregations(@Nonnull String path) {
-    final String includeFilter = ESUtils.escapeReservedCharacters(path) + "/.*";
-    final String excludeFilter = ESUtils.escapeReservedCharacters(path) + "/.*/.*";
+    final String currentLevel = ESUtils.escapeReservedCharacters(path) + "/.*";
+    final String nextLevel = ESUtils.escapeReservedCharacters(path) + "/.*/.*";
+    final String nextNextLevel = ESUtils.escapeReservedCharacters(path) + "/.*/.*/.*";
```

Comment:
isn't this still vulnerable to the case where there are too many paths `n` levels down? is there any way to protect against that?

Reply:
If the next level has too many, the 10000 limit will help alleviate the load. We do need this since not having this check caused bugs before where an entity appears as a group